src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/MilestoneController.kt

```diff
@@ -91,8 +93,11 @@ class MilestoneController {
         if (existingMilestone.startDate != updateData.startDate || existingMilestone.endDate != updateData.endDate) {
             updateTaskEstimations(existingMilestone, milestoneTasks, updateData.startDate, updateData.endDate, userId)
         }
+
         existingMilestone.startDate = updateData.startDate
         existingMilestone.endDate = updateData.endDate
+        existingMilestone.originalStartDate = updateData.startDate
+        existingMilestone.originalEndDate = updateData.endDate
```

Comment:
```suggestion
        existingMilestone.originalStartDate = updateData.originalStartDate
        existingMilestone.originalEndDate = updateData.originalEndDate
```
Why not update it with the same fields?